packages/core/src/App/Components/Layout/Header/dtrader-v2/real-signup-banner-dtrader-v2.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import classNames from 'classnames';
+import 'Sass/app/_common/components/banner-dtrader-v2.scss';
+import { Text } from '@deriv-com/quill-ui';
+import {
+    CurrencyBtcIcon,
+    CurrencyEthIcon,
+    CurrencyEurIcon,
+    CurrencyUsdIcon,
+    LabelPairedChevronRightMdRegularIcon,
+} from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+
+type TRealSignupBannerDTraderV2 = {
+    openRealAccount: () => void;
+    is_eu?: boolean;
+};
+const RealSignupBannerDTraderV2 = ({ openRealAccount, is_eu }: TRealSignupBannerDTraderV2) => (
+    <button className='banner__container' onClick={openRealAccount}>
+        <div className='banner'>
+            <div
+                className={classNames('banner__currency', {
+                    'banner__currency--is-eu': is_eu,
+                })}
+            >
+                {is_eu ? (
+                    <React.Fragment>
+                        <CurrencyEurIcon iconSize='md' className='icon-1' />
+                        <CurrencyUsdIcon iconSize='md' className='icon-2' />
+                    </React.Fragment>
+                ) : (
+                    <React.Fragment>
+                        <CurrencyBtcIcon iconSize='md' className='icon-1' />
+                        <CurrencyEthIcon iconSize='md' className='icon-2' />
+                        <CurrencyUsdIcon iconSize='md' className='icon-3' />
```

Comment:
Can we use CSS selectors such as :nth-child(2), :nth-child(3) in .scss instead of adding these 'icon-2', 'icon-3' classnames?
We can add `banner__icon` className to add some common properties that are common between these icons (`position`).
However, for specific styles that differ, it seems better to utilize `:nth-child()`.
What do you think? )

Reply:
Of course we can, I was just trying to avoid where it's possible and use classNames (in order to stick to BEM). Will apply